Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -46,7 +46,22 @@
         return m_geometrySink.Get();
     }
 
-    _CGPathCustomSink(_In_ ID2D1GeometrySink* sink) : m_geometrySink(sink), m_lastPoint{ 0, 0 }, m_isFigureOpen(false) {
+    _CGPathCustomSink(_In_ ID2D1GeometrySink* sink)
+        : m_geometrySink(sink),
+          m_lastPoint{ 0, 0 },
+          m_isFigureOpen(false),
+          m_allowsFigureCalls(false),
+          m_hasGeometryStarted(false),
+          m_lastClose(D2D1_FIGURE_END_CLOSED),
+          m_allowsFinalFigureCall(false),
+          m_delayedEndFigure(false) {
+    }
+
+    STDMETHOD_(void, _delayedEndFigureCall)() {
```

Comment:
>STDMETHOD_ [](start = 4, length = 10)

This doesnt have to be STDMETHOD_() because this is not part of the interface ID2D1GeometrySink